sw/device/lib/dif/dif_keymgr.c

```diff
@@ -518,8 +493,8 @@ dif_result_t dif_keymgr_generate_versioned_key(
 
 dif_result_t dif_keymgr_sideload_clear_set_enabled(const dif_keymgr_t *keymgr,
                                                    dif_toggle_t state) {
-  bool enable = false;
-  if (keymgr == NULL || !toggle_to_bool(state, &enable)) {
+  if (keymgr == NULL ||
+      (state != kDifToggleEnabled && state != kDifToggleDisabled)) {
```

Comment:
Can we haz `is_valid_toggle()`?

Reply:
Done.

(I actually made this a function in `dif_base` too since it is reused between IP DIF libs).